lib/wcgi-runner/tests/integration.rs

```diff
@@ -0,0 +1,95 @@
+use std::path::Path;
+
+use bytes::Bytes;
+use http::{Request, StatusCode};
+use hyper::{body::HttpBody, Body};
+use tempfile::TempDir;
+use wcgi_host::CgiDialect;
+use wcgi_runner::Builder;
+
+const FERRIS_SAYS: &str = "https://registry-cdn.wapm.dev/packages/wasmer-examples/ferris-says/ferris-says-0.2.0-2f5dfb76-77a9-11ed-a646-d2c429a5b858.webc";
```

Comment:
At the very least we shouldn't depend on a hard CDN path, but rather use `REGISTRY.com/namespace/package@version` with `Accept: application/webc`.

Reply:
Done.